src/jest.js

```diff
@@ -47,7 +47,15 @@ function getTestPaths(testRunner, config, patternInfo) {
   if (patternInfo.onlyChanged) {
     return findOnlyChangedTestPaths(testRunner, config);
   } else {
-    return testRunner.promiseTestPathsMatching(new RegExp(patternInfo.pattern));
+    return new Promise((resolve, reject) => {
+      fs.access(patternInfo.pattern, (err) => {
```

Comment:
woah, why are you using `fs.access`, I didn't even know it existed! I feel like `fs.existsSync` should be used here. There is no reason really to do this operation async – we are blocked on the result on it anyway.